cmd/operator/main.go

```diff
@@ -92,6 +93,7 @@ func init() {
 	flag.Var(&flSteps, "step", "a percentage in traffic the candidate should go through")
 	flag.StringVar(&flStepsString, "steps", "5,20,50,80", "define steps in one flag separated by commas (e.g. 5,30,60)")
 	flag.Int64Var(&flInterval, "interval", 0, "the time between each rollout step")
+	flag.Uint64Var(&flMinRequestCount, "min-requests", 0, "expected minimum requests before determining candidate's health")
```

Comment:
Lets set a default value

Reply:
Let me research and think of a potential default value

Reply:
I've been thinking about this, but it becomes hard to set a default value that works for many (most) services. This value really depends on the traffic your service often receives, which might require some analysis of the traffic in the past (which is possible). Also, this value also depends on the decision in #43 

If we decide to do #43, it might be easier to go with some "random" number like 1000.

Reply:
Per #43 I think we should go with default=0 (no such requirement).